content/runner/aws/installation.md

```diff
@@ -40,65 +40,63 @@ The AWS runner is configured using environment variables. This article reference
 - Setup up vpc firewall rules for the build instances [ec2 authorizing-access-to-an-instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html) We need allow ingress and egress access to port 9079. Once complete you will have a security group id, which is needed for configuration of the runner.
 - (optional) For debugging purposes, you can amend the security group with the following rules:
 
   - `SSH TCP 22   0.0.0.0/0` for linux.
   - `RDP TCP 3389 0.0.0.0/0` for windows.
 
-  This will allow you to remotely connect to the build instances. Once you set `DRONE_SETTINGS_AWS_KEY_PAIR_NAME`.
-
-## AWS EC2 environment variables
-
-Set up the runner by using either an environment variables or a `.env` file similar to other Drone runners. Below is a list of the AWS specific environment variables.
-
-- __DRONE_SETTINGS_AWS_ACCESS_KEY_ID__
-  : AWS access key id, obtained above.
-- __DRONE_SETTINGS_AWS_ACCESS_KEY_SECRET__
-  : AWS access key secret, obtained above.
-- __DRONE_SETTINGS_AWS_REGION__
-  : AWS region
-- __DRONE_SETTINGS_AWS_AVAILABILITY_ZONE__
-  : Specify the availability zone for the build instance.
-- __DRONE_SETTINGS_AWS_KEY_PAIR_NAME__
-  : The name of the key pair to use for the build instances. This is optional and used for debugging purposes.
-- __DRONE_SETTINGS_REUSE_POOL__
-  : Reuse existing ec2 instances on restart of the runner.
-- __DRONE_SETTINGS_LITE_ENGINE_PATH__
+  This will allow you to remotely connect to the build instances. Once you set `key_pair_name`.
+
+## Pool specific environment variables
+
+Set up the runner by using either an environment variables or a `.env` file similar to other Drone runners. Below is a list of the specific environment variables.
+
+- __DRONE_REUSE_POOL__
+  : Reuse existing instances on restart of the runner.
+- __DRONE_LITE_ENGINE_PATH__
   : The web URL for the path containing lite-engine binaries. This can be hosted internally or you can get the binaries from github.
 
+  If you are moving from an older version, some change to setup may be required, they are outlined [here]({< relref "version migration" >}}).
+
 ## Example AWS Runner configuration `.env` file
 
 {{< highlight bash "linenos=table" >}}
 DRONE_RPC_HOST=localhost:8080
 DRONE_RPC_PROTO=http
 DRONE_RPC_SECRET=bea26a2221fd8090ea38720fc445eca6
-DRONE_SETTINGS_AWS_ACCESS_KEY_ID=XXXXXX
-DRONE_SETTINGS_AWS_ACCESS_KEY_SECRET=XXXXX
-DRONE_SETTINGS_AWS_REGION=us-east-2
-DRONE_SETTINGS_LITE_ENGINE_PATH=https://github.com/harness/lite-engine/releases/download/v0.0.1.12/
+DRONE_LITE_ENGINE_PATH=https://github.com/harness/lite-engine/releases/download/v0.0.1.12/
```

Comment:
can we remove lite-engine path from the installation since it now has a default value? I suspect most customers won't need to worry about this in their initial setup